packages/plugins/deploy-tracker/src/trackingFunctions.js

```diff
@@ -111,6 +111,44 @@ export default class TrackingFunctions {
     cb();
   }
 
+  async trackMigrationScript(params, callback) {
+    if (!this.enabled) {
+      return callback();
+    }
+
+    const {
+      scriptName,
+      scriptDirectory,
+      forceTracking,
+      migrationsDirectory
+    } = params;
+
+    if (!forceTracking && scriptDirectory !== migrationsDirectory) {
+      return callback();
+    }
+
+    const { hash } = await this.block;
+    const chains = await this.chains;
+
+    if (!chains[hash]['migrations']) {
```

Comment:
Is it possible that `chains[hash]` would be undefined too? If so, it would nee to be initiated as `{}`